### PR TITLE
replace irate to rate to allow see volatile spikes

### PR DESCRIPTION
**Pull request by Slach**
Signed-off-by: Slach <bloodjazman@gmail.com>

### CODE REVIEW

**Review by alex-zaitsev — APPROVED** [review]

**Comment by filimonov** [comment]
> see volatile spikes

isn't it the opposite? I think irate (instant-rate) shows the volatile spikes better

https://medium.com/@kavyaprathyusha/rate-vs-irate-in-promql-a172e3d9c38f ?

File: grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json [hunk 1]
```diff
@@ -916,14 +916,14 @@
...
         {
-          "expr": "irate(chi_clickhouse_event_SelectQuery{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"}[1m])",
+          "expr": "rate(chi_clickhouse_event_SelectQuery{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"}[1m])",
           "intervalFactor": 2,
           "legendFormat": "select {{hostname}}",
           "refId": "B",
           "step": 10
         },
         {
-          "expr": "irate(chi_clickhouse_event_Query{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"}[1m])",
```
Comment (filimonov) [inline grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json:926]: [1m]

maybe it would be better to use $_rate_interval?

**Comment by filimonov** [comment]
BTW: do you use https://github.com/grafana/dashboard-linter ?

**Comment by Slach** [comment]
no, we don't use it. will try to add it in our github actions

File: grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json (hunk 1 above)
Comment (Slach) [reply grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json:926]: ok. let's try

Comment (Slach) [reply grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json:926]: thanks for suggestions

**Pull request merged**